src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
 
 ![Response Threshold Widget.](dashboard-response-threshold-widget.png)
 
-### Top issues
+### My top issues
 
-You can track the impact of your issues by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter for error events:
+You can track the impact of issues assigned to you or your team by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by updating the search condition to filter for issue assignment:
 
-- Data Set: `All Events`
-- Search condition: `event.type:error`
+- Search condition: `is:unresolved is:for_review assigned_or_suggested:me`
 
-Update columns to group by `issues` and apply `count()` as the aggregating function. Order count in descending order so the widget displays issues with the most error events:
+Update columns to add `project` so you can see the associated project with each issue, and modify the “sort by” to `Events` so the widget displays issues with the most error events in descending order:
```

Comment:
I think the quote characters (") here have a different code than the standard quotes we use
```suggestion
Update columns to add `project` so you can see the associated project with each issue, and modify the "sort by" to `Events` so the widget displays issues with the most error events in descending order:
```